packages/ui/atomic/health-check/src/index.spec.ts

```diff
@@ -0,0 +1,95 @@
+// import {startVerdaccio} from '@coveo/verdaccio-starter';
+import {spawnSync} from 'node:child_process';
+import {copyFileSync} from 'fs-extra';
```

Comment:
Prefer `cpSync` over copyFileSync, better sys call. (you may need recursive and force true)